### ISSUE TITLE
[Results Summary] Duplicated styles

### ISSUE THREAD

**Issue by BitBeaver-182** [peter-stefanski/FrontendMentror#2]
https://github.com/peter-stefanski/FrontendMentror/blob/8cfaa120f9bd7096180bddb2878b92c4c68b3220/Results%20summary%20component1/results-summary-component-main/index.html#L10-L21

On line 10 you are importing styles.css
https://github.com/peter-stefanski/FrontendMentror/blob/8cfaa120f9bd7096180bddb2878b92c4c68b3220/Results%20summary%20component1/results-summary-component-main/index.html#L10

But on lines 12-21 you are declaring a new styles tag. Is there a reason for this? If not, it's best to add it to the styles.css file you had

https://github.com/peter-stefanski/FrontendMentror/blob/8cfaa120f9bd7096180bddb2878b92c4c68b3220/Results%20summary%20component1/results-summary-component-main/index.html#L12-L21

**Comment by peter-stefanski** [comment]
yeah that was from "Frontend Mentor" when I've downloaded that file I didn't see that styles lines, sorry Sensei. I've changed that misktakes
